resources/assets/js/app.js

```diff
@@ -277,6 +277,28 @@ $(function() {
         $this.next('.group-items').toggleClass('hide');
     });
 
+    $('.select-group').on('click', function () {
+        var $this = $(this);
+        var $parentGroup = $this.closest('ul.list-group');
+        $parentGroup.find('input[type=checkbox]').prop('checked', true);
+
+        $parentGroup.find('.group-items').removeClass('hide')
+        $parentGroup.find('.group-toggle').addClass('ion-ios-minus-outline').removeClass('ion-ios-plus-outline');
+	event.stopPropagation();
+        return false;
+    });
+
+    $('.deselect-group').on('click', function () {
+        var $this = $(this);
+        var $parentGroup = $this.closest('ul.list-group');
```

Comment:
Since you're only referencing `$this` once, you may as well just make it `$(this).closest(...)`.